assets/rich_text_components_definitions.ts

```diff
@@ -102,8 +102,12 @@ export default {
       "name": "url",
       "description": "The link URL. If no protocol is specified, HTTPS will be used.",
       "schema": {
-        "type": "custom",
-        "obj_type": "SanitizedUrl"
+        "type": "unicode",
+        "obj_type": "SanitizedUrl",
+        "validators": [{
+          "id": "is_regex_matched",
+          "regex_pattern": "(^https:\\/\\/.*)|(^(?!.*:\\/\\/)(.*))"
```

Comment:
Just wondering, where did we get this RegEx pattern from? Did you write it yourself or find it somewhere? It works really well.